src/node_buffer.cc

```diff
@@ -760,6 +761,54 @@ void ReadDoubleBE(const FunctionCallbackInfo<Value>& args) {
 
 
 template <typename T, enum Endianness endianness>
+void ReadInt64Generic(const FunctionCallbackInfo<Value>& args,
+                      const char* formatter) {
+  Environment* env = Environment::GetCurrent(args);
+
+  THROW_AND_RETURN_UNLESS_BUFFER(Environment::GetCurrent(args), args[0]);
+  SPREAD_BUFFER_ARG(args[0], ts_obj);
+
+  uint32_t offset = args[1]->Uint32Value();
```

Comment:
Can you use the overload that takes a `context` argument?

Reply:
This is copied from https://github.com/nodejs/node/blob/1b7372f2fb55f704b885e1097e2ec0381068c855/src/node_buffer.cc#L724

I could fix it in ReadInt64Generic and make it inconsistent, or fix it in both places as part of this PR, or make a separate PR to fix it in ReadFloatGeneric.